data/sql/updates/pending_db_world/Fix_quest_11879.sql

```diff
@@ -0,0 +1,60 @@
+-- Kaw events
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 2580200) AND (`source_type` = 9) AND (`id` IN (0, 1, 2, 3, 4));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2580200, 9, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 1, 0, 2000, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Yell'),
+(2580200, 9, 1, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 59, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Enable Running'),
+(2580200, 9, 2, 0, 0, 0, 100, 1, 3000, 3000, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, 3974.17, 5476.31, 35.602, 5.564, 'Kaw - Timed - Move to Moria'),
+(2580200, 9, 3, 0, 0, 0, 100, 1, 2500, 2500, 0, 0, 0, 11, 46260, 2, 0, 0, 0, 0, 19, 25881, 100, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Mount to Moria'),
+(2580200, 9, 4, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 19, 33024, 0, 0, 0, 0, 0, 19, 25881, 100, 0, 0, 0, 0, 0, 0, 'Kaw - Remove Moria Flags Immune To Players');
+-- Make her NON_ATTACKABLE until the event starts
+UPDATE `creature_template` SET `unit_flags` = 0 WHERE (`entry` = 25881);
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 25881);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(25881, 0, 0, 0, 101, 0, 100, 1, 1, 10, 0, 0, 0, 0, 80, 2588100, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Moria - On Min Players in Range - Run Script (No Repeat)'),
+(25881, 0, 1, 0, 6, 0, 100, 1, 0, 0, 0, 0, 0, 0, 19, 33024, 0, 0, 0, 0, 0, 19, 25802, 100, 0, 0, 0, 0, 0, 0, 'Moria - On Just Died - Remove Flags Immune To Players (No Repeat)'),
+(25881, 0, 2, 0, 1, 0, 100, 1, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Moria - Out of Combat - Set Reactstate Passive (No Repeat)'),
+(25881, 0, 3, 0, 1, 0, 100, 1, 0, 0, 0, 0, 0, 0, 28, 46260, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Moria - Out of Combat - Remove Aura \'Riding Wooly Mammoth Bull\' (No Repeat)'),
+(25881, 0, 4, 0, 1, 0, 100, 1, 0, 0, 0, 0, 0, 0, 18, 256, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Moria - On Just Died - Set Flags Immune To Players (No Repeat)'),
```

Comment:
Remove Flags Immune To Players is repeated, also see above, flags aren't added anyways

Reply:
in fact, these flags are for Moria. Maybe the comment is wrong. :cry:

Reply:
Oh I read it wrong, my bad. But something's fishy with Moria's flags. It should be set On Respawn. Can Swim doesn't do much, so it doesn't need to be set on script, and it's missing Immune To NPCs so there'd be trouble with pets